### DISCUSSION TITLE
Running compiled code outside Visual Studio

### DISCUSSION THREAD

**Discussion by fruit-boots** [VictorGordan/opengl-tutorials#13] [Q&A] [answered]
Hey all, I'm not sure if I'm doing something wrong, but for some reason, after I compile/build the projects I can't run the .exe file. When building and running in  local windows debugger everything seems to be working fine. Is there an additional step that I'm missing?

**Comment by VictorGordan** [comment, marked as answer]
You need to place resources (shaders, textures, models, etc) in the
directory of the exe file. Keep in mind they need to have the same relative
path as in the directory of the VS project. ;)

**Reply by fruit-boots** [reply]
Perfect, works like a charm, thank you!!
